usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -63,13 +63,18 @@ def test_sles_hardened(host, get_release_value, is_sles_sap, is_sle_micro):
     )
 
     result = host.run(
-        "sudo oscap xccdf eval --local-files {dir} --profile {profile} {file}".format(
+        "sudo oscap xccdf eval --report /var/tmp/report.html --local-files {dir} --profile {profile} {file}".format(
```

Comment:
I'm seeing a significant increase in test time with the report included. If a test is run on an ephemeral instance this report will be lost anyway. Is there reason enough to be dumping this report by default? Rather than something done manually on a failure?

The test is already quite slow at 70-80 seconds. With the report run time is tripling to 210+ seconds.

Reply:
> I'm seeing a significant increase in test time with the report included. If a test is run on an ephemeral instance this report will be lost anyway. Is there reason enough to be dumping this report by default? Rather than something done manually on a failure?
> 
> The test is already quite slow at 70-80 seconds. With the report run time is tripling to 210+ seconds.

This report is _useful_ for us in openQA to report bugs.  It's not lost to us as we retrieve it.

We can reach a compromise to create this report only if an option is passed to img-proof or if an environment variable is set.  WDYT?

Reply:
I don't want to optimize prematurely but in the case this command fails in our workflow we would run the test 3 times. That means possibly ~700 seconds spent on one test. That would be compared to previous max of 210.

It'd be possible to pass through a flag, the trouble will be not coupling img-proof to the sles test suite. I.e. we cannot add a `--include-hardened-report` to img-proof. It'd have to be done in an agnostic way.

Reply:
> It'd be possible to pass through a flag, the trouble will be not coupling img-proof to the sles test suite. I.e. we cannot add a `--include-hardened-report` to img-proof. It'd have to be done in an agnostic way.

What about an environment variable like `SCAP_REPORT=/var/tmp/report.html`.  That way we'd also have control over the path.

Reply:
> That could work. So the idea would be that in your workflow you set the env var and the test only creates report if that var is set in the test instance?

Yes. Will begin work on it right now.

Reply:
> That could work. So the idea would be that in your workflow you set the env var and the test only creates report if that var is set in the test instance?

It's working now:

https://openqa.suse.de/tests/13332663/file/img_proof-report.html

Please merge and release.